Ginger/GingerCore/ALM/RQM/ImportFromRQM.cs

```diff
@@ -1004,7 +1006,7 @@ public static ObservableList<ExternalItemFieldBase> GetOnlineItemFields(Backgrou
                     }
 
                     //Parallel computing solution
-                    List<XmlNode> entryList = new List<XmlNode>();
+                    List<XmlNode> entryList = new();
                     if (categoryTypeUriPages.Count > 1)
                     {
```

Comment:
Please check if code can be refactored as most of the code block in if is duplicated in else block as well